app.yaml

```diff
@@ -0,0 +1,6 @@
+runtime: python37
+automatic_scaling:
+  min_instances: 1
+  max_instances: 100
+  min_idle_instances: 1
+entrypoint: gunicorn -b :$PORT main:app --worker-class sanic.worker.GunicornWorker
```

Comment:
What is the value of $PORT here? 
The app uses the default host 127.0.0.1 and port 8000. Do we need to make these configurable as part of deploying?

Reply:
Yup. I should learn more about how this works. I lifted this from a stackoverflow post. I assume that `$PORT` is an environment variable that app engine provides. I should confirm that.

My understanding is that the default host and port 8000 are only used when it's invoked as a main program, i.e. `python main.py`, which is not the case when served from gunicorn.

I'll do some reading to make sure my understanding is correct.

Reply:
`gunicorn -b :$PORT main:app` is straight from Google's docs: https://cloud.google.com/appengine/docs/standard/python3/runtime#application_startup. We're just adding the `worker-class` option to bridge between gunicorn and sanic.